core/templates/dev/head/pages/exploration_editor/editor_tab/issues_overview.html

```diff
@@ -0,0 +1,18 @@
+<div ng-controller="IssuesOverview">
+  <span style="font-size: 16px;">
+    <strong>Issues Overview</strong>
+  </span>
+
+  <md-card style="background-color: white; margin: 20px 0px; padding: 8px;">
```

Comment:
Perhaps any edits to an answer group should trigger a refresh? If we want to do that in a separate PR then a temporary "fix" for now might be to add, in small grey text, "Last refreshed at {{time}}" somewhere on this card.